api/server/index.js

```diff
@@ -97,21 +68,55 @@ config.validate(); // Validate the config
   app.use('/api/plugins', routes.plugins);
   app.use('/api/config', routes.config);
 
-  // static files
+  // Static files
   app.get('/*', function (req, res) {
     res.sendFile(path.join(projectPath, 'dist', 'index.html'));
   });
 
   app.listen(port, host, () => {
     if (host == '0.0.0.0') {
       console.log(
-        `Server listening on all interface at port ${port}. Use http://localhost:${port} to access it`,
+        `Server listening on all interfaces at port ${port}. Use http://localhost:${port} to access it`,
       );
     } else {
       console.log(`Server listening at http://${host == '0.0.0.0' ? 'localhost' : host}:${port}`);
     }
   });
-})();
+};
+
+const configureSocialLogins = (app) => {
```

Comment:
if this is being factored out as a function, it can go in a separate file.

Reply:
I have tested with Google, Github, and Discord, and the login works.